instances/tissueSampleAttribute/mounted.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/tissueSampleAttribute/mounted",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/TissueSampleAttribute",
+  "definition": "A specimen that has been preserved in a way that can be stored and handled easily for an extended period of time (e.g., on a microscope slide with mounting medium and covered with a coverslip).",
```

Comment:
I don't like this one... although it often is includes preservation in histology, "mounted" does not necessarily mean it is preserved and can be handled on an extended period of time. You can also mount fresh specimen and then throw them away. Maybe that definition is more accurate (feel free to adapt though):

"A specimen is typically 'mounted' on a clear glass slide with aqueous or non-aqueous mounting media in order to be supported for further handling. Often 'mounted' specimen are additionally covered with a coverslip, especially for long term preservation."

Reply:
Good suggestion. I shortened it a bit, though.